cffi/cdefs.h

```diff
@@ -936,6 +955,9 @@ LY_ERR lys_parse(struct ly_ctx *, struct ly_in *, LYS_INFORMAT, const char **, s
 LY_ERR ly_ctx_new_ylpath(const char *, const char *, LYD_FORMAT, int, struct ly_ctx **);
 LY_ERR ly_ctx_get_yanglib_data(const struct ly_ctx *, struct lyd_node **, const char *, ...);
 
+LY_ERR lydict_insert(const struct ly_ctx *, const char *, size_t, const char **);
+LY_ERR lydict_remove(const struct ly_ctx *, const char *);
```

Comment:
Same comment, could you add this in the commit where it is used?

Reply:
Exactly the same thing as the other one, not used directly by libyang-python. This API is needed if you want to play with parsed tree and not cause segfaults on C layer. The problem behind is that you can use direct `str2c` pointers in those places, but you cannot control when those strings will be released and in which order as you can only destroy whole context, not unload individual modules. And this destroy process is exactly expecting strings to be in the context dictionary. So that's why I need those API to be called by my APP.

Reply:
Let's wait for @rjarry's answer to the other comment, as it is the same problem.